operator/controllers/model_initializer_injector_test.go

```diff
@@ -35,7 +35,7 @@ func TestStorageInitalizerInjector(t *testing.T) {
 	_, err = mi.InjectModelInitializer(&d, containerName, "gs://mybucket/mymodel", "", "", "")
 	g.Expect(err).To(BeNil())
 	g.Expect(len(d.Spec.Template.Spec.InitContainers)).To(Equal(1))
-	g.Expect(d.Spec.Template.Spec.InitContainers[0].Image).To(Equal("kfserving/storage-initializer:v0.6.1"))
+	g.Expect(d.Spec.Template.Spec.InitContainers[0].Image).To(Equal("seldonio/rclone-storage-initializer:1.16.0"))
```

Comment:
That should probably be `1.16.0-dev` here. We also need to make sure that this will be updated by the release script.

Reply:
This is just a test string - part of the unit tests - it could have just `foo` in there.